src/tags.ts

```diff
@@ -93,7 +93,10 @@ export const parseFooterTagLines = (
 ): _.Dictionary<string> => {
 	return _.chain(footerTagLines)
 		.reject(line => {
-			return _.isEmpty(line.trim());
+			return (
```

Comment:
@robertgzr LGTM one little NIT would be to only .trim() once and then test both conditions on the result. But its mostly aesthetical